cmd/admin-handlers-users.go

```diff
@@ -399,14 +399,26 @@ func (a adminAPIHandlers) AddUser(w http.ResponseWriter, r *http.Request) {
 			AccountName:     parentUser,
```

Comment:
Also, it is weird that we rely on parent user to check if it is allowed or not to set credentials. This means that a child can change the credentials of its sibling